src/sgame/sg_cm_world.cpp

```diff
@@ -444,16 +460,22 @@ void G_CM_LinkEntity( gentity_t *gEnt )
 	// set the abs box
 	if ( gEnt->r.bmodel && ( angles[ 0 ] || angles[ 1 ] || angles[ 2 ] ) )
 	{
-		// expand for rotation
-		float max;
+		vec3_t matrix[3];
```

Comment:
So the idea is that it used to be OK for G_CM_LinkEntity to use any upper bound of the actual boundaries, but since stuff is being used for bots it needs to be as precise a bound as we can get?

Reply:
Precisely